index.html

```diff
@@ -53,7 +53,255 @@ <h3 class="link-item">
+            </div>
+            <div class="project-subtitle-container">
+              <ul class="subtitle-items">
+                <li class="list-less"><p>CANOPY</p></li>
+                <li class="list"><p>Back End Dev</p></li>
+                <li class="list"><p>2015</p></li>
+              </ul>
+            </div>
+            <div class="project-article-container">
+              <p class="description-item description-item-b">
+                A daily selection of privately perosonalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="project-tag-container">
+              <ul class="tag-item">
+                <li class="tags"><a class="tag-btn" href="#">html</a></li>
+                <li class="tags"><a class="tag-btn" href="#">css</a></li>
+                <li class="tags"><a class="tag-btn" href="#">javaScript</a></li>
+              </ul>
+            </div>
+            <div class="projects-btn">
+              <input type="button" value="See Project" class="project-button" />
+            </div>
+          </div>
+         </div>
+        </div>
+        <div class="content work-projects work-projects-b">
+          <div class="item project-header">
+            <div class="projectsimage-c">
+              <img
+                src="assets/images/portfolio-image4.PNG"
+                alt="project image title"
+                class="headimage-b"
+              />
+            </div>
+          </div>
+          <div class="cards-description">
+          <div class="item project-description">
+            <div class="project-title-container">
+              <h4 class="description-title">Tonic</h4>
+            </div>
+            <div class="project-subtitle-container">
+              <ul class="subtitle-items">
+                <li class="list-less"><p>CANOPY</p></li>
+                <li class="list"><p>Back End Dev</p></li>
+                <li class="list"><p>2015</p></li>
+              </ul>
+            </div>
+            <div class="project-article-container">
+              <p class="description-item description-item-b">
+                A daily selection of privately perosonalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="project-tag-container">
+              <ul class="tag-item">
+                <li class="tags"><a class="tag-btn" href="#">html</a></li>
+                <li class="tags"><a class="tag-btn" href="#">css</a></li>
+                <li class="tags"><a class="tag-btn" href="#">javaScript</a></li>
+              </ul>
+            </div>
+            <div class="projects-btn">
+              <input type="button" value="See Project" class="project-button" />
+            </div>
+          </div>
+         </div>
+        </div>
+      </section>
+    <section id="about-myself">
+        <div class="content about-project">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_10](https://user-images.githubusercontent.com/69638013/158978681-314bccfd-c7b1-43b8-beb5-77862e543c9f.png)  | ![Screenshot_11](https://user-images.githubusercontent.com/69638013/158978798-025d4b81-eda0-4a1d-811c-24214a96318b.png)

- [ ] Kindly add the background border and the right icon in languages section check the screenshot above